src/component/App/App.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import './App.css';
```

Comment:
- [ ] I noticed a constant mixture of components and styling files, kindly create a folder outside the components that stores your CSS files and import them into their respective components.

Reply:
Please show me docs saying so because I watch a lot of respectful programmers like you, sir they add the CSS file inside the component so please give docs or anything saying so.